### PR TITLE
Fix 5406 and 4970 progress indicator LogCacheActivity / LogTrackableActivity

### PR DESCRIPTION
**Pull request by kumy**
While digging for fixing those two issues, I've found that things around `ActionBarListActivity` could be factorized. I've started by doing that, then fix 5406&4970  issues.
Do you agree with such changes?

### CODE REVIEW

File: main/src/cgeo/geocaching/activity/AbstractActivity.java [hunk 1]
```diff
@@ -139,6 +135,9 @@ protected void onCreate(final Bundle savedInstanceState, @LayoutRes final int re
     }
 
     private void initializeCommonFields() {
```
Comment (rsudev) [inline main/src/cgeo/geocaching/activity/AbstractActivity.java]: If you re-purpose the method, you should change its name as well

**Comment by rsudev** [comment]
I support such refactoring activities in general, but was today merely 'flying over' your changes here. For some more comments from my side, I need another more thorough look through the changes.

**Comment by kumy** [comment]
@rsudev I've updated the PR according to your remark about renaming the function.

**Comment by kumy** [comment]
@rsudev or others, any objection to merge into master?

**Pull request merged**